docs/content/users/quickstart.md

```diff
@@ -318,6 +318,20 @@ In the web browser, log into your account using `admin` and `publish`.
 
 Visit [Ibexa documentation](https://doc.ibexa.co/en/latest/getting_started/install_with_ddev/) for more cases.
 
+## Joomla
+
+```bash
+mkdir my-joomla-site && cd my-joomla-site
+tag=$(curl -L "https://api.github.com/repos/joomla/joomla-cms/releases/latest" | docker run -i --rm ddev/ddev-utilities jq -r .tag_name) && curl -L "https://github.com/joomla/joomla-cms/releases/download/$tag/Joomla_$tag-Stable-Full_Package.zip" -o joomla.zip
+unzip ./joomla.zip && rm joomla.zip
+ddev config --project-type=php --webserver-type=apache-fpm --upload-dirs=images
+ddev start
+ddev php installation/joomla.php install --site-name="My Joomla Site" --admin-user="John Doe" --admin-username=admin --admin-password=AdminAdmin1! --admin-email=admin@mail.com --db-type=mysql --db-encryption=0 --db-host=db --db-user=db --db-pass="db" --db-name=db --db-prefix=ddev_ --public-folder=""
```

Comment:
Why not use "Administrator" or "Administrator Name" as the user real name instead of "John Doe"? I love the John Doe reference but it is probably easier for users when there are no such references.

Reply:
Agreed. 
```suggestion
ddev php installation/joomla.php install --site-name="My Joomla Site" --admin-user="Administrator" --admin-username=admin --admin-password=AdminAdmin1! --admin-email=admin@mail.com --db-type=mysql --db-encryption=0 --db-host=db --db-user=db --db-pass="db" --db-name=db --db-prefix=ddev_ --public-folder=""
```